packages/functions/utils/scheduler/getScheduleExtension.ts

```diff
@@ -3,13 +3,19 @@ import utc from "dayjs/plugin/utc";
 
 dayjs.extend(utc);
 
-export const getScheduleWithDaysOffset = (daysToAdd: number): string => {
+export type TimeOffset = {
+  days?: number;
+  months?: number;
+};
+
+export const getScheduleWithTimeOffset = (offset: TimeOffset): string => {
```

Comment:
This can something like, offset:number, type:string, and the type can be 'months' | 'days' just like how the add() works, and dont have to use double add, WDYT?

Reply:
I guess both are fine, yours is maybe a bit easier to extend for other units (like week or minute)